src/components/shared/FilterNavigatorBar.css

```diff
@@ -102,6 +102,14 @@
   border-color: rgba(0, 0, 0, 0.2);
 }
 
+.filterNavigatorBarUncommittedItem {
+  opacity: 0.75;
+}
```

Comment:
I think the opacity is still too high; can you try with 0.65 ?

Also, like we discussed in a separate commit, I think we can try to animate going from this class to the commited value. I think we can animate on both `opacity` and `color`. Please do this in a separate commit :)